### ISSUE TITLE
DNN library is not found.

### ISSUE THREAD

**Issue by Abhijeet241093** [sankethsj/seatbelt-detection#1]
Detected at node 'sequential_4/sequential_1/model1/Conv1/Conv2D' defined at (most recent call last):
    File "main.py", line 75, in <module>
      y_pred, score = prediction_func(img_crop)
    File "main.py", line 30, in prediction_func
      pred = predictor.predict(img)
    File "/home/creapple/anaconda3/envs/seatbelt_detection_py/lib/python3.8/site-packages/keras/utils/traceback_utils.py", line 65, in error_handler
      return fn(*args, **kwargs)
    File "/home/creapple/anaconda3/envs/seatbelt_detection_py/lib/python3.8/site-packages/keras/engine/training.py", line 2382, in predict
      tmp_batch_outputs = self.predict_function(iterator)
    File "/home/creapple/anaconda3/envs/seatbelt_detection_py/lib/python3.8/site-packages/keras/engine/training.py", line 2169, in predict_function
      return step_function(self, iterator)
    File "/home/creapple/anaconda3/envs/seatbelt_detection_py/lib/python3.8/site-packages/keras/engine/training.py", line 2155, in step_function
      outputs = model.distribute_strategy.run(run_step, args=(data,))
    File "/home/creapple/anaconda3/envs/seatbelt_detection_py/lib/python3.8/site-packages/keras/engine/training.py", line 2143, in run_step
      outputs = model.predict_step(data)
    File "/home/creapple/anaconda3/envs/seatbelt_detection_py/lib/python3.8/site-packages/keras/engine/training.py", line 2111, in predict_step
      return self(x, training=False)
    File "/home/creapple/anaconda3/envs/seatbelt_detection_py/lib/python3.8/site-packages/keras/utils/traceback_utils.py", line 65, in error_handler
      return fn(*args, **kwargs)
    File "/home/creapple/anaconda3/envs/seatbelt_detection_py/lib/python3.8/site-packages/keras/engine/training.py", line 558, in __call__
      return super().__call__(*args, **kwargs)
    File "/home/creapple/anaconda3/envs/seatbelt_detection_py/lib/python3.8/site-packages/keras/utils/traceback_utils.py", line 65, in error_handler
      return fn(*args, **kwargs)
    File "/home/creapple/anaconda3/envs/seatbelt_detection_py/lib/python3.8/site-packages/keras/engine/base_layer.py", line 1145, in __call__
      outputs = call_fn(inputs, *args, **kwargs)
    File "/home/creapple/anaconda3/envs/seatbelt_detection_py/lib/python3.8/site-packages/keras/utils/traceback_utils.py", line 96, in error_handler
      return fn(*args, **kwargs)
    File "/home/creapple/anaconda3/envs/seatbelt_detection_py/lib/python3.8/site-packages/keras/engine/sequential.py", line 412, in call
      return super().call(inputs, training=training, mask=mask)
    File "/home/creapple/anaconda3/envs/seatbelt_detection_py/lib/python3.8/site-packages/keras/engine/functional.py", line 512, in call
      return self._run_internal_graph(inputs, training=training, mask=mask)
    File "/home/creapple/anaconda3/envs/seatbelt_detection_py/lib/python3.8/site-packages/keras/engine/functional.py", line 669, in _run_internal_graph
      outputs = node.layer(*args, **kwargs)
    File "/home/creapple/anaconda3/envs/seatbelt_detection_py/lib/python3.8/site-packages/keras/utils/traceback_utils.py", line 65, in error_handler
      return fn(*args, **kwargs)
    File "/home/creapple/anaconda3/envs/seatbelt_detection_py/lib/python3.8/site-packages/keras/engine/training.py", line 558, in __call__
      return super().__call__(*args, **kwargs)
    File "/home/creapple/anaconda3/envs/seatbelt_detection_py/lib/python3.8/site-packages/keras/utils/traceback_utils.py", line 65, in error_handler
      return fn(*args, **kwargs)
    File "/home/creapple/anaconda3/envs/seatbelt_detection_py/lib/python3.8/site-packages/keras/engine/base_layer.py", line 1145, in __call__
      outputs = call_fn(inputs, *args, **kwargs)
    File "/home/creapple/anaconda3/envs/seatbelt_detection_py/lib/python3.8/site-packages/keras/utils/traceback_utils.py", line 96, in error_handler
      return fn(*args, **kwargs)
    File "/home/creapple/anaconda3/envs/seatbelt_detection_py/lib/python3.8/site-packages/keras/engine/sequential.py", line 412, in call
      return super().call(inputs, training=training, mask=mask)
    File "/home/creapple/anaconda3/envs/seatbelt_detection_py/lib/python3.8/site-packages/keras/engine/functional.py", line 512, in call
      return self._run_internal_graph(inputs, training=training, mask=mask)
    File "/home/creapple/anaconda3/envs/seatbelt_detection_py/lib/python3.8/site-packages/keras/engine/functional.py", line 669, in _run_internal_graph
      outputs = node.layer(*args, **kwargs)
    File "/home/creapple/anaconda3/envs/seatbelt_detection_py/lib/python3.8/site-packages/keras/utils/traceback_utils.py", line 65, in error_handler
      return fn(*args, **kwargs)
    File "/home/creapple/anaconda3/envs/seatbelt_detection_py/lib/python3.8/site-packages/keras/engine/training.py", line 558, in __call__
      return super().__call__(*args, **kwargs)
    File "/home/creapple/anaconda3/envs/seatbelt_detection_py/lib/python3.8/site-packages/keras/utils/traceback_utils.py", line 65, in error_handler
      return fn(*args, **kwargs)
    File "/home/creapple/anaconda3/envs/seatbelt_detection_py/lib/python3.8/site-packages/keras/engine/base_layer.py", line 1145, in __call__
      outputs = call_fn(inputs, *args, **kwargs)
    File "/home/creapple/anaconda3/envs/seatbelt_detection_py/lib/python3.8/site-packages/keras/utils/traceback_utils.py", line 96, in error_handler
      return fn(*args, **kwargs)
    File "/home/creapple/anaconda3/envs/seatbelt_detection_py/lib/python3.8/site-packages/keras/engine/functional.py", line 512, in call
      return self._run_internal_graph(inputs, training=training, mask=mask)
    File "/home/creapple/anaconda3/envs/seatbelt_detection_py/lib/python3.8/site-packages/keras/engine/functional.py", line 669, in _run_internal_graph
      outputs = node.layer(*args, **kwargs)
    File "/home/creapple/anaconda3/envs/seatbelt_detection_py/lib/python3.8/site-packages/keras/utils/traceback_utils.py", line 65, in error_handler
      return fn(*args, **kwargs)
    File "/home/creapple/anaconda3/envs/seatbelt_detection_py/lib/python3.8/site-packages/keras/engine/base_layer.py", line 1145, in __call__
      outputs = call_fn(inputs, *args, **kwargs)
    File "/home/creapple/anaconda3/envs/seatbelt_detection_py/lib/python3.8/site-packages/keras/utils/traceback_utils.py", line 96, in error_handler
      return fn(*args, **kwargs)
    File "/home/creapple/anaconda3/envs/seatbelt_detection_py/lib/python3.8/site-packages/keras/layers/convolutional/base_conv.py", line 290, in call
      outputs = self.convolution_op(inputs, self.kernel)
    File "/home/creapple/anaconda3/envs/seatbelt_detection_py/lib/python3.8/site-packages/keras/layers/convolutional/base_conv.py", line 262, in convolution_op
      return tf.nn.convolution(
Node: 'sequential_4/sequential_1/model1/Conv1/Conv2D'
DNN library is not found.
	 [[{{node sequential_4/sequential_1/model1/Conv1/Conv2D}}]] [Op:__inference_predict_function_6381]

**Comment by sankethsj** [comment]
Hello @Abhijeet241093, please check the README file for detailed instructions on how to setup and run this script.
